lambda/template.yaml

```diff
@@ -18,29 +11,15 @@ Resources:
   IcebergMetricsLambda:
     Type: AWS::Serverless::Function
     Properties:
-      CodeUri: lambda/
-      Handler: app.lambda_handler
-      Runtime: python3.9
+      PackageType: Image
+      ImageUri: !Ref ImageURL
       Policies: 
         - CloudWatchPutMetricPolicy: {}
         - AWSLambdaBasicExecutionRole
         - AmazonS3ReadOnlyAccess
+        - ECR-Pull-Read-Only
```

Comment:
Hey @itakserman-cloudinary ,
I am getting CloudFormation error when trying to deploy. It looks like you're using _ECR-Pull-Read-Only_ policy which isn't defined in a SAM. There are pre-configured SAM policies that we can reuse, if the one you suggesting isn't there you will need to add relevant permissions as separate statement in the template.yaml.

See the list of SAM policies here:
https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-policy-templates.html